gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -571,6 +573,28 @@
 
   private static final String PEGASUS_PLUGIN_CONFIGURATION = "pegasusPlugin";
 
+  // Enable the use of generic pegasus schema compatibility checker
+  private static final String ENABLE_PEGASUS_SCHEMA_COMPATIBILITY_CHECK = "pegasusPlugin.enablePegasusSchemaCompatibilityCheck";
+
+  private static final String PEGASUS_SCHEMA_SNAPSHOT = "PegasusSchemaSnapshot";
+
+  private static final String PEGASUS_EXTENSION_SCHEMA_SNAPSHOT = "PegasusExtensionSchemaSnapshot";
+
+  private static final String PEGASUS_SCHEMA_SNAPSHOT_DIR = "pegasusSchemaSnapshot";
```

Comment:
I feel they are two different use cases. Extension schema snapshot is a default feature, regular one is an optional feature. It could be clearer to keep them into separate directories. Also whether there will be a scenario regular schema and extension schema have the same name coincidently?